docs/samples/v1beta1/triton/bert/README.md

```diff
@@ -0,0 +1,171 @@
+
+# QA Inference with BERT model using Triton Inference Server
+Bidirectional Embedding Representations from Transformers (BERT), is a method of pre-training language representations which obtains state-of-the-art results on a wide array of Natural Language Processing (NLP) tasks.
+
+This example demonstrates
+- Inference on Question Answering (QA) task with BERT Base/Large model
+- The use of fine-tuned NVIDIA BERT models
+- Deploy Transformer for preprocess using BERT tokenizer
+- Deploy BERT model on Triton Inference Server
+- Inference with V2 KFServing protocol
+
+We can run inference on a fine-tuned BERT model for tasks like Question Answering.
+
+Here we use a BERT model fine-tuned on a SQuaD 2.0 Dataset which contains 100,000+ question-answer pairs on 500+ articles combined with over 50,000 new, unanswerable questions.
+
+## Setup
+1. Your ~/.kube/config should point to a cluster with [KFServing 0.5 installed](https://github.com/kubeflow/kfserving/#install-kfserving).
```

Comment:
Above it said to use version 0.4

Reply:
That's the existing doc for KFS 0.4, this example is new for KFS 0.5(v1beta1) which uses triton v2 protocol.